src/modules/python/pbs/v1/_base_types.py

```diff
@@ -1067,7 +1066,8 @@ def increment_chunks(self, increment_spec):
                             num = chunk_ct + inc
                         elif isinstance(increment_dict[i], str):
                             if increment_dict[i].endswith('%'):
-                                p_inc = float(increment_dict[i][:-1])/100 + 1.0
+                                p_inc = float(
```

Comment:
I didn't do this intentionally, but pydev did it to be PEP compliant because the line was > 80 characters, and I think we'll need to keep it this way otherwise travis will fail now